### ISSUE TITLE
Project is abandoned :(

### ISSUE THREAD

**Issue by lefuglyduck** [HuskyDG/magisk-files#181]
Sadly this project appears abandoned. It no longer works with the latest grapheneos (based on Android 15). And issue after issue appears with no responses. Go elsewhere folks. If anyone has a clue how to get root on the latest version of grapheneos, let us know.

**Comment by lefuglyduck** [comment]
I'm aware I can root but unfortunately not having zygisk is a deal breaker. It's needed for banking apps.

**Comment by DarkenLX** [comment]
> I'm aware I can root but unfortunately not having zygisk is a deal breaker. It's needed for banking apps.

has anyone tried reaching out to husky on telegram?

**Comment by 1q23lyc45** [comment]
I'm sorry to hear that ......
I would suggest, however, that it's time to switch to APatch; Kitsune Mask is, after all, at the user-space level, and certainly won't be able to keep up with the effects of APatch at the kernel level. As for the Shamiko module, I highly discourage its use. It is very insecure.

https://github.com/1q23lyc45/kitsunemask
This is my modified source code where I fixed a security issue and updated some dependencies, anyone who wants to continue maintaining Kitsune Mask can fork it. But please be sure to follow the GPLv3 license. All forms of closed source are not allowed.

**Comment by gaestur** [comment]
There is another actively developed fork of Magisk that supports GrapheneOS, which may work for you: https://github.com/pixincreate/Magisk (I have some problems with modules there, but I suspect my hands much more than the project :) )

**Comment by 1q23lyc45** [comment]
> > I'm sorry to hear that ...... I would suggest, however, that it's time to switch to APatch; Kitsune Mask is, after all, at the user-space level, and certainly won't be able to keep up with the effects of APatch at the kernel level. As for the Shamiko module, I highly discourage its use. It is very insecure.
> > https://github.com/1q23lyc45/KitsuneMagisk This is my modified source code where I fixed a security issue and updated some dependencies, anyone who wants to continue maintaining Kitsune Mask can fork it. But please be sure to follow the GPLv3 license. All forms of closed source are not allowed.
> 
> i thought kitsune was closed source, how come you can find its source code?, repo just went public right?

To clarify:
Magisk Delta, including Kitsune Mask, is an open source project that follows GPLv3 open source. The original repository is publicly available at https://github.com/KitsuneMagisk/magisk. However, the source code is only in the kitsune branch. master branch is upstream 27001 source code.

The one that isn't open source is the Alpha version, and I've never successfully booted any device using the Alpha version.

**Comment by gaestur** [comment]
@lefuglyduck An update: ver 28002 of github.com/pixincreate/Magisk in Graphene 2024120200 works, Zygisk works, modules work (if needed), several banking apps work, Google Wallet doesn't work even with PlayIntegrityFork (initially it shows a device as compliant, but does some re-check when adding a card and fails).

**Comment by 1q23lyc45** [comment]
I've seen a ghost, strange.
https://github.com/HuskyDG/magisk-files/tree/main/images
Last commit date is yesterday? WHAT?

**Comment by VendorAttestation** [comment]
Move to https://github.com/bmax121/APatch its basically undetected and supports 99% magisk modules

**Comment by 1q23lyc45** [comment]
> Move to https://github.com/bmax121/APatch its basically undetected and supports 99% magisk modules

I agree with you. Kitsune Mask is actually only for devices that support neither KernelSU nor APatch. I'll admit that Kitsune Mask, as a program that runs at the user level, can't do the same thing as it does at the kernel level.

**Comment by lefuglyduck** [comment]
@VendorAttestation, but does it support GrapheneOS?

**Comment by 1q23lyc45** [comment]
> @VendorAttestation, but does it support GrapheneOS?

Did you run fastboot flashing unlock after installing grapheneos? Also, is your device "OEM unlock" switch turned on?
If you haven't already, go back up your data first and then unlock your device's bootloader.

Theoretically, all devices that have Bootloader unlocked and have a Linux kernel version between 3.18-6.1 are supported.

**Comment by 1q23lyc45** [comment]
> > Move to https://github.com/bmax121/APatch its basically undetected and supports 99% magisk modules
> 
> No, it is bad, even with pekaboo module

Why? APatch is much better than Kitsune Mask.
Are you using the latest CI version of APatch? It seems like you're not.

**Comment by VendorAttestation** [comment]
> @VendorAttestation, but does it support GrapheneOS?

i'd assume so if kenel

**Comment by Jefferderp** [comment]
> @lefuglyduck An update: ver 28002 of github.com/pixincreate/Magisk in Graphene 2024120200 works, Zygisk works, modules work (if needed), several banking apps work, Google Wallet doesn't work even with PlayIntegrityFork (initially it shows a device as compliant, but does some re-check when adding a card and fails).

This worked like a charm! Thank you.



> Move to https://github.com/bmax121/APatch its basically undetected and supports 99% magisk modules

Unfortunately APatch bootloops when I try to boot the modified boot.img. Perhaps I did something wrong, but I don't think so.

**Comment by VendorAttestation** [comment]
> > @lefuglyduck An update: ver 28002 of github.com/pixincreate/Magisk in Graphene 2024120200 works, Zygisk works, modules work (if needed), several banking apps work, Google Wallet doesn't work even with PlayIntegrityFork (initially it shows a device as compliant, but does some re-check when adding a card and fails).
> 
> This worked like a charm! Thank you.
> 
> > Move to https://github.com/bmax121/APatch its basically undetected and supports 99% magisk modules
> 
> Unfortunately APatch bootloops when I try to boot the modified boot.img. Perhaps I did something wrong, but I don't think so.

latest nightly build?

**Comment by 1q23lyc45** [comment]
[Download APatch Nightly(CI) Build](https://nightly.link/bmax121/APatch/workflows/build/main/APatch.zip)

**Comment by Jefferderp** [comment]
> latest nightly build?

No, the latest APatch stable release, 10762 on May 1st.

Is the nightly specifically known to solve this problem?

**Comment by 1q23lyc45** [comment]
> > latest nightly build?
> 
> No, the latest APatch stable release, 10762 on May 1st.
> 
> Is the nightly specifically known to solve this problem?

Do NOT use "stable" version.
The latest stable version is outdated and has a privilege escalation vulnerability. Please make sure to use the CI version.
[Download APatch CI Release](https://nightly.link/bmax121/APatch/workflows/build/main/APatch.zip)

**Comment by G2Split** [comment]
Should I still use Kitsune Mask? People say there are issues, but I haven't experienced any as far as I remember, even though I've been using it since early 2024. I'm on a community GSI (SuperiorOS), Android 13, and everything is working fine for me.
